### PR TITLE
Workflow queue

### PR DESCRIPTION
**Pull request by omichel**
This PR implements a workflow queue for each competition to avoid a git conflict in the `participants.txt` file in case two competition workflows are running in parallel.

The [run workflow](https://github.com/cyberbotics/competition-template/blob/112ce269a2c97fa11be7f5300b7868c88e9e2b26/.github/workflows/run.yml#L61-L64) of the competition template was updated to notify webots.cloud when the job is complete and the queue item can be released, and a new job can possibly start.

### CODE REVIEW

File: httpdocs/run_competition.php [hunk 1]
```diff
@@ -81,15 +82,67 @@ function check_participant_token($github_token, $participant_repo) {
...
+if ($total == 0)
+  $running = 'R:';
+else
+  $running = '';
+$query = "INSERT IGNORE INTO queue(project, participant) VALUES((SELECT id FROM project WHERE url LIKE 'https://github.com/$organizer/%'), '$running$participant')";
+$mysqli->query($query) or die($mysqli->error);
+
+if ($total == 0)
+  repository_dispatch($organizer, $participant);
+elseif $mysqli->affected_rows == 1
```
Comment (Jean-Eudes-le-retour) [inline httpdocs/run_competition.php]: When I tested my own competition I got:
```
<b>Parse error</b>:  syntax error, unexpected variable &quot;$mysqli&quot;, expecting &quot;(&quot; in <b>/home/clients/102a72498d695e3ad1163abedff6c1d5/webots.cloud/main/httpdocs/run_competition.php</b> on line <b>144</b><br />
```
It seems like there are parentheses missing
```suggestion
elseif ($mysqli->affected_rows == 1)
```

**Review by Jean-Eudes-le-retour — CHANGES_REQUESTED** [review]
I got an error when I did the self-test for my competition.

[Action logs](https://github.com/Jean-Eudes-le-retour/pit_escape_competition/actions/runs/3738341904/jobs/6344335750)

**Review by Jean-Eudes-le-retour — APPROVED** [review]
I've tested the queue and everything seems to work now.

Thank you!

**Comment by omichel** [comment]
Thank you!

**Pull request merged**